### PR TITLE
feat: add volcnao-global scheduler

### PR DESCRIPTION
**Pull request by Vacant2333**
add the volcano-global scheduler, based on the karmada scheduler, the scheduelr will implement gang scheduling later

### CODE REVIEW

**Comment by guoqinwill** [comment]
Is the go.mod file missing?

**Comment by wangyang0616** [comment]
Whether the compilation and running of the current pr meet the expectation？

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,130 @@
+module volcano.sh/volcano-global
+
+go 1.21.10
```
Comment (Monokaix) [inline go.mod:3]: keep this with karmada is ok and no need to add toolchain.

**Comment by Vacant2333** [comment]
> Whether the compilation and running of the current pr meet the expectation？

yes, the compliation is ok now, running needs the deployment, will raise another pr for all other contents

File: go.mod (hunk 1 above)
Comment (Vacant2333) [reply go.mod:3]: The Go version of the Karmada version we use is consistent with our current, thanks!

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+	gomodules.xyz/jsonpatch/v2 v2.4.0
+	k8s.io/api v0.29.4
+	k8s.io/apimachinery v0.29.4
+	k8s.io/client-go v0.29.4
+	k8s.io/component-base v0.29.4
+	k8s.io/klog/v2 v2.120.1
+	k8s.io/kube-openapi v0.0.0-20240228011516-70dd3763d340
+	sigs.k8s.io/controller-runtime v0.17.5
+	volcano.sh/apis v1.9.0
+	volcano.sh/volcano v1.9.0
```
Comment (Monokaix) [inline go.mod:17]: Change to v1.10 is better and what's the latest k8s version does Karmada support?

Comment (Monokaix) [reply go.mod:17]: If it's not used we should exec `go mod tidy` to remove them: )

**Pull request closed without merging**